server.js

```diff
@@ -13,6 +15,19 @@ io.on('connection', (socket) => {
   })
 })
 
+app.post('/', (req, resp) => {
```

Comment:
what is `'/'` in reference to this line?

Reply:
The is the equivalent to the home-page. It's the 'ending', or everything that follows the `url`.

Reply:
Gotcha. So we wouldn't need to update it for things that are getting sent to and from /gamepage? I suppose the Gamepage component would send that action back up to App anyways

Reply:
Sort of. The actual `fetch` has it's url set already, and you can call that from any compenent at any path, and it will always access the same `url`.

TL;DR: client-url !== server-url

Reply:
![gotcha](https://media.giphy.com/media/3oz8xs8WYZr0dJV3kQ/giphy.gif)